### PR TITLE
release v1.0.1

### PR DESCRIPTION
**Pull request by rg911**
change log and version update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+## [1.0.1] - 24-May-2021
+
+**Milestone**: Symbol Mainnet
+ Package  | Version  | Link
+---|---|---
+SDK Core| v1.0.1 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
+Catbuffer | v1.0.0 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
+Client Library | v1.0.0  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
+
+- Fxied missing `utf-8` dependancy issue.
```
Comment (fboucquez) [inline CHANGELOG.md:16]: Typo in Fxied

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ The Symbol SDK for TypeScript / JavaScript allows you to develop web, mobile, an
 
 ### _Catapult-Server_ Network Compatibility (catapult-server@0.10.0.8)
 
-Symbol network pre-launch [catapult-server](https://github.com/nemtech/catapult-server/releases/tag/v0.10.0.8), **it is recommended to use this package's 0.23.3 version and upwards for the upcoming pre-release versions and final release**.
+Symbol network launched [catapult-server](https://github.com/nemtech/catapult-server/releases/tag/v1.0.0), **it is recommended to use this package's 1.0.0 version and upwards for the upcoming pre-release versions and final release**.
```
Comment (fboucquez) [inline README.md:14]: link is https://github.com/nemtech/catapult-server/releases/tag/v1.0.0.0 (4 digits version)

File: README.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The Symbol SDK for TypeScript / JavaScript allows you to develop web, mobile, an
 
 ### _Catapult-Server_ Network Compatibility (catapult-server@0.10.0.8)
 
-Symbol network launched [catapult-server](https://github.com/nemtech/catapult-server/releases/tag/v1.0.0), **it is recommended to use this package's 1.0.0 version and upwards for the upcoming pre-release versions and final release**.
+Symbol network launched [catapult-server](https://github.com/nemtech/catapult-server/releases/tag/v1.0.00), **it is recommended to use this package's 1.0.0 version and upwards for the upcoming pre-release versions and final release**.
```
Comment (fboucquez) [inline README.md:14]: Symbol network launched [catapult-server](https://github.com/nemtech/catapult-server/releases/tag/v1.0.0.0), **it is recommended to use this package's 1.0.0.0 version and upwards for the upcoming pre-release versions and final release**.

1.0.0.0 may change, I've asked the new version

**Review by fboucquez — CHANGES_REQUESTED** [review]

**Review by fboucquez — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+## [1.0.1] - 24-May-2021
+
+**Milestone**: Symbol Mainnet
+ Package  | Version  | Link
+---|---|---
+SDK Core| v1.0.1 | [symbol-sdk](https://www.npmjs.com/package/symbol-sdk)
+Catbuffer | v1.0.0 | [catbuffer-typescript](https://www.npmjs.com/package/catbuffer-typescript)
+Client Library | v1.0.0  | [symbol-openapi-typescript-fetch-client](https://www.npmjs.com/package/symbol-openapi-typescript-fetch-client)
+
+- Fixed missing `utf-8` dependancy issue.
```
Comment (OlegMakarenko) [inline CHANGELOG.md:16]:
```suggestion
- Fixed missing `utf-8` dependency issue.
```

**Review by OlegMakarenko — CHANGES_REQUESTED** [review]

**Pull request merged**